src/bibliotheca/policy/upgrades/upgrades.py

```
@@ -96,3 +96,73 @@ def install_kimug(context):
+                setattr(fti, attr, value.replace(old_prefix, new_prefix))
+                fixed_types.append(type_id)
+
+    # 2. Invalidate the Dexterity schema cache so corrected schemas re-resolve.
+    try:
+        from plone.dexterity.schema import SCHEMA_CACHE
+
+        for type_id in fixed_types:
+            SCHEMA_CACHE.invalidate(type_id)
+        SCHEMA_CACHE.clear()
+    except Exception:  # pragma: no cover - best effort
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)